src/action.js

```diff
@@ -12,6 +11,13 @@ const defaults = {
 
 async function action(context = github.context) {
   try {
+
+    var customConfigFile = '.github/pr-labeler.yml'; // default path of config file
```

Comment:
If configuration path is provided with the `with` parameter, we can use `core.getInput('configuration-path')`:

```js
const configPath = core.getInput('configuration-path', { required: true })
```